src/Initializer/Parameters/ModelParameters.cpp

```diff
@@ -72,6 +96,10 @@ ModelParameters readModelParameters(ParameterReader* baseReader) {
                          tv,
                          boundaryFileName,
                          materialFileName,
-                         itmParameters};
+                         itmParameters,
+#ifdef USE_DAMAGEDELASTIC
+    damagedElasticParameters
```

Comment:
Indentation is off.

Reply:
Also, we could remove the ifdef, we can just pass an empty struct, if we use another model.

Reply:
Modified and removed the ifdefs